src/utils/HypervBoxUtil.js

```diff
@@ -0,0 +1,61 @@
+import fs from 'fs';
+import path from 'path';
+import util from './Util';
+import Promise from 'bluebird';
+
+var HypervBox = {
+  command: function () {
+    let hypervPath = "C:\Program Files\Hyper-V";
+    if (util.isWindows()) {
+        if (HypervBox.pathExists())
+        {
+            return hypervPath;
+        }
```

Comment:
imo this function needs to return an exe file name, which is used below, to check the version